packages/eslint-config-terra/eslint.config.js

```diff
@@ -66,7 +66,7 @@ module.exports = {
       },
     },
     {
-      files: ['**/wdio/**/*-spec.*'],
+      files: ['**/wdio/**/*-spec.*', '**/full-stack/**/*-spec.*'],
```

Comment:
Are we for sure going with full-stack for this concept? You had mentioned potentially wanting to renaming this a week or so ago

Reply:
ok, thinking about this more, there are enough unknowns to warrant me closing this

yeah, i discussed changing the name to `integration-testing`, but i backed away from that. in the future, full-stack could potentially be used for application tests, not just engines -  at that point, `system-testing` seems like a better fit. this is also dependent on how we scope terra-toolkit v6. let's revisit this further down the road